### PR TITLE
Added solution. Added .gitignore

### PR DESCRIPTION
**Pull request by JovannyM**

### CODE REVIEW

**Review by localconst — COMMENTED** [review]
You, my comrade, have made a piece of shit here. Please clean up after yourself. Thank you!

**Comment by JovannyM** [comment]
> You, my comrade, have made a piece shit here. Please clean up after yourself. Thank you!

What you said about my mom?

**Comment by localconst** [comment]
> > You, my comrade, have made a piece shit here. Please clean up after yourself. Thank you!
> 
> What you said about my mom?

Clearly nothing

File: Program.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+namespace Bubelinez
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
```
Comment (rumyax) [inline Program.cs]:
```suggestion
}

```

Comment (localconst) [reply Program.cs]: Awesome

**Review by rumyax — APPROVED** [review]
L G T M !

**Pull request merged**